CRM/MembersOnlyEvent/Service/MembersOnlyEventAccessService.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+use CRM_MembersOnlyEvent_BAO_EventMembershipType as EventMembershipType;
+use CRM_MembersOnlyEvent_Configurations as Configurations;
+use CRM_MembersOnlyEvent_BAO_MembersOnlyEvent as MembersOnlyEvent;
+
+class CRM_MembersOnlyEvent_Service_MembersOnlyEventAccessService {
```

Comment:
I meant to change MembersOnlyEventAccess to MembersOnlyEventAccessService only in the the other places that uses it, but the class name itself should remain as it used to be, the reason is that the class already contain the keyword "service" in it `_Service_` , so no need to duplicate that in the name again, but when you changed the class alias like that : 

`use CRM_MembersOnlyEvent_Service_MembersOnlyEventAccessService as MembersOnlyEventAccess;`

you had to include the keyword "service" to "MembersOnlyEventAccess" to make it clear that it is the service class since we are not using the full name of the class.